nkms/crypto/keyring/keys/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .signing_keypair import SigningKeypair
```

Comment:
I'm not a huge fan of this pattern.  Why not put both of these classes in `nkms/crypto/keypairs.py`?  Cleaner namespacing; no awkward import in `__init__` - don't get me wrong, plenty of packages do this, but I think it's just confusing.